### PR TITLE
Python3 and pep8 updates

### PR DESCRIPTION
**Pull request by matthewduren**
Update both functions to support python3 and more closely follow pep8 style. Tested working on local.

### CODE REVIEW

File: pricing_import.py [hunk 1]
```diff
@@ -1,419 +1,443 @@
-#!/usr/bin/env python2
-
+#!/usr/local/bin/python
```
Comment (Bjorn248) [inline pricing_import.py]: Isn't using env more flexible than having a static path to python here?

Comment (matthewduren) [reply pricing_import.py]: Probably...I had modified that with the intent of including a docker-compose.yml and a python3 container.  Reverted.

File: pricing_import.py [hunk 2]
```diff
@@ -1,419 +1,443 @@
 #!/usr/bin/env python2
```
Comment (Bjorn248) [inline pricing_import.py]: Wait you went back to a python2 shebang?! lol wut

Comment (Bjorn248) [reply pricing_import.py]: Isn't this python 3? Or is it both 2 and 3?

**Pull request merged**